lib/dynamo-helpers.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const _ = require('lodash');
+
+
+/**
+ * Prepare key replacements (ExpressionAttributeNames) for a dynamo expression
+ *
+ * @param {array} fields - list of fields or column names
+ * @return {object}
+ *
+ * @example
+ * prepareAttributeNames(['plan', 'status']);
+ * // returns
+ * {
+ *   '#plan': 'plan',
+ *   '#status': 'status'
+ * }
+ */
+module.exports.prepareAttributeNames = (fields) => {
+  const keys = _.map(fields, (field) => `#${field}`);
+  return _.zipObject(keys, fields);
+};
+
+
+/**
+ * Prepare value replacements (ExpressionAttributeValues) for a dynamo expression
+ *
+ * @param {object} values - hash map of values
+ * @return {object}
+ *
+ * @example
+ * prepareAttributeValues({ plan: 'free1', status: 'active' });
+ * // returns
+ * {
+ *   ':plan': 'free1',
+ *   ':status': 'active'
+ * }
+ */
+module.exports.prepareAttributeValues = (values) => {
+  return _.mapKeys(values, (value, key) => `:${key}`);
+};
+
+
+/**
+ * Prepare a Dynamo UpdateExpression
+ *
+ * @param {object|array} updates
+ * @return {string}
+ *
+ * @example
+ * prepareUpdateExpression({ plan: 'free1', status: 'active' });
+ * // returns
+ * 'SET #plan = :plan AND #status = :status'
+ *
+ */
+module.exports.prepareUpdateExpression = (updates) => {
```

Comment:
Yeah actually thinking maybe we should just accept the keys array, and call into this with `Object.keys()`.  That seems like it might make the most sense if it was used outside of this module directly